stdlib/public/core/VarArgs.swift

```diff
@@ -290,7 +290,9 @@ final public class VaListBuilder {
     // supported vararg type is greater than the alignment of Int.
     // FIXME: this implementation is not portable because
     // alignof differs from the ABI alignment on some architectures
-#if os(watchOS) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
+    // FIXME: This is valid for AAPCS calling convention, FreeBSD/arm
+    // may need this too.
+#if (os(watchOS) || os(Linux)) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
```

Comment:
iOS does not include tvOS, but there's no 32-bit tvOS. I think we should make normal AAPCS the default and opt out for Apple platforms rather than the other way around.

I guess you're right about not touching non-ARM platforms for now.